apps/pie-storybook/decorators/component-status.ts

```diff
@@ -25,9 +26,24 @@ export const ComponentStatus = (story: () => TemplateResult, storybookContext: S
         <div class="pie-sb-status-wrapper">
             ${story()}
 
-            <pie-tag variant="${tagVariant}" class="${`c-componentStatus ${positionClass}`}">
-                Status: ${componentStatus}
-            </pie-tag>
+            <div class="${`c-componentStatus ${positionClass}`}">
+                <pie-tag variant="${tagVariant}">
+                    Status: ${componentStatus}
+                </pie-tag>
+
+                <pie-tag>
```

Comment:
why tag the icon?

Reply:
Valid question, I'll add a screenshot to see what you prefer the look of